build/README.md

```diff
@@ -520,3 +520,13 @@ existing clusters you will need to:
     instances with appropriate values where applicable as explained in the above section.
 
 1.  Run `tk apply workspace/$CLUSTER_CONTEXT_schema_manager`
+
+### Database version and server cache
+Current version: 3.1.0
+Minimum support version: 3.0.0
+
+Changes in v3.1.0
+* Added a new writer field to identification_service_areas and subscriptions table. 
+* Read the database version and cache it in memory. 
+
+If you start server with database v3.0.0. and migrate database to v3.1.0 without restarting the server, the service will work, the writer field will be ignored. However, if you start the server with database v3.1.0 and you downgrade the database to 3.0.0, the service will fail because the server will try to write/update the writer field
```

Comment:
We want to make sure the consumer of this documentation understands the expected behavior of the system at the level they're interested in.  The audience for this documentation is someone who is deploying a DSS, but may not be intimately familiar with how it works.  So, they likely would not know what a writer field is, what it does, or what the implications of it being ignored are.  If correct, we should say something like "...the service will work normally but undeleted, expired ISAs will not be automatically removed from the database by the system, even when the garbage collection feature is finished and enabled."